src/library/scala/collection/convert/DecorateAsJava.scala

```diff
@@ -18,91 +18,91 @@ import scala.language.implicitConversions
 trait DecorateAsJava extends AsJavaConverters {
   /**
    * Adds an `asJava` method that implicitly converts a Scala `Iterator` to a Java `Iterator`.
```

Comment:
I don't think it's necessary for trivial changes. For real changes, we try to remember to update as we go, but often fail. We don't really have any good story for this.